lib/Twig/Node/Expression/Call.php

```diff
@@ -239,9 +239,10 @@ private function getCallableParameters($callable, $isVariadic)
             if ($argument && $argument->isArray() && $argument->isDefaultValueAvailable() && array() === $argument->getDefaultValue()) {
                 array_pop($parameters);
             } else {
-                $callableName = $r->name;
-                if ($r->getDeclaringClass()) {
-                    $callableName = $r->getDeclaringClass()->name.'::'.$callableName;
+                if ($r instanceof ReflectionMethod) {
+                    $callableName = $r->class.'::'.$r->name;
+                } else {
+                    $callableName = $r->name;
```

Comment:
This should be moved back where it was before. This was done on purpose so that the `$callableName` is always defined (for HHVM/Hack).